ui/app/AppLayouts/Profile/views/ContactsView.qml

```diff
@@ -67,222 +64,155 @@ SettingsContentBase {
             }
         }
 
-        StackLayout {
-            id: stackLayout
-            anchors.left: parent.left
-            anchors.right: parent.right
-            anchors.top: contactsTabBar.bottom
-            currentIndex: contactsTabBar.currentIndex
-            anchors.topMargin: Theme.padding
-            // CONTACTS
-            ColumnLayout {
-                Layout.fillWidth: true
-                Layout.minimumHeight: 0
-                Layout.maximumHeight: (verifiedContacts.height + mutualContacts.height + noFriendsItem.height)
-                visible: (stackLayout.currentIndex === 0)
-                onVisibleChanged: {
-                    if (visible) {
-                        stackLayout.height = height+contactsTabBar.anchors.topMargin;
-                    }
-                }
-                spacing: Theme.padding
-                ContactsListPanel {
-                    id: verifiedContacts
-
-                    Layout.fillWidth: true
-                    title: qsTr("Trusted Contacts")
-                    visible: !noFriendsItem.visible && count > 0
-                    contactsModel: root.mutualContactsModel
-                    searchString: searchBox.text
-                    onOpenContactContextMenu: root.openContextMenu(contactsModel, publicKey)
-                    panelUsage: Constants.contactsPanelUsage.verifiedMutualContacts
-                    onSendMessageActionTriggered: {
-                        root.contactsStore.joinPrivateChat(publicKey)
-                    }
-                }
-
-                ContactsListPanel {
-                    id: mutualContacts
-
-                    Layout.fillWidth: true
-                    visible: !noFriendsItem.visible && count > 0
-                    title: qsTr("Contacts")
-                    contactsModel: root.mutualContactsModel
-                    searchString: searchBox.text
-                    onOpenContactContextMenu: root.openContextMenu(contactsModel, publicKey)
-                    panelUsage: Constants.contactsPanelUsage.mutualContacts
-
-                    onSendMessageActionTriggered: {
-                        root.contactsStore.joinPrivateChat(publicKey)
-                    }
-                }
-
-                Item {
-                    id: noFriendsItem
-
-                    Layout.fillWidth: true
-                    Layout.preferredHeight: visible ?  (root.contentHeight - (2*searchBox.height) - contactsTabBar.height - contactsTabBar.anchors.topMargin) : 0
-                    visible: root.mutualContactsModel.ModelCount.empty
+        SearchBox {
+            id: searchBox
+            Layout.fillWidth: true
+            placeholderText: qsTr("Search by name or chat key")
+        }
+    }
 
-                    NoFriendsRectangle {
-                        anchors.centerIn: parent
-                        text: qsTr("You don't have any contacts yet")
-                    }
-                }
+    ContactsListPanel {
```

Comment:
When the user changes tab and go back to the previous one, the state is lost, because all the tabs share the same list. This behavior is not expected by the user and should be avoided.

As a solution we can use StackLayout with 3 independent instances of `ContactsListPanel`. As the lists are no longer "unrolled" it's still clean and performant solution.

Moreover having 3 instances will simplify code because `panelUsage` is no longer needed. It means much less imperative code resolving parameters based on the current value of `panelUsage`. 

`section.labelPositioning` can be moved to `ContactsListPanel` as it's common for all cases.

`header` can be defined as a `Component` and shared by all 3 lists.